Issues #3 y #4

RLNicolas opened the pull request:
Se modifico con bootstrap la barra de navegacion, ademas tambien se cambio en las paginas de presentacion, index y chat el boton publicar proyecto, con el cual se presentaba un problema ya que habia en dos css un estilo con el mismo nombre (btn-publicar).

RLNicolas committed:
Issues #3 y #4

Se modifico con bootstrap la barra de navegacion, ademas tambien se cambio en las paginas de presentacion, index y chat el boton publicar proyecto, con el cual se presentaba un problema ya que habia en dos css un estilo con el mismo nombre (btn-publicar).

Santo1215 commented on public/Estilos/presentacionEstilos.css:
Excelente integración pero comprobar si es posible, copiar y pegar la clase de proyectos estilos de btn-publicar, ya que al hacer el link con proyectos estilos cambian algunos detalles de la pagina

![image](https://github.com/user-attachments/assets/22cafe58-7958-4cd0-b541-4b7bc8520a38)

Santo1215 commented on public/Pagina/index.html:
Ver si se pueden reducir un poco los margenes de la izquierda

![image](https://github.com/user-attachments/assets/17867a9d-ca69-4dd0-b801-c8c127303117)

Santo1215 commented on public/Pagina/perfil.html:
Excelentes mejoras +10 puntos

Santo1215 commented on public/presentacion.html:
Mismo comentario que presentacionestilos

Santo1215 reviewed (changes requested):
Ver si es posible continuar con las observaciones para proceder con el merge, falta revisión de Sebastián en los archivos trabajados por el

RLNicolas committed:
Correccion observaciones Pull Request

Se corrigio el link de los html a sus respectivos css, tambien se corrigieron los margenes en index y chat. Revisar en la pagina de presentacion que en funcion del zoom de la pagina puede llegar a variar la alineacion de las imagenes

RLNicolas commented:
En el commit mas reciente se realizaron las respectivas correcciones a las observaciones

Santo1215 reviewed (approved):
Excelente, se puede continuar con el merge y cerrando los issues
